docs/readmes/basics/quick_start_guide.md

```diff
@@ -214,7 +226,8 @@ NMS can take upto 60 seconds to finish starting up.
 You will probably want to enable this organization (magma-test) to access all networks,
 so go to [host.localhost](https://host.localhost) and login with the same credentials.
 Once there, you can click on the "Organizations" tab on the left sidebar, choose
-`magma-test`, and then check "Enable all networks" in the subsequent pop-up window.
+`magma-test` (the three dots on the right) &rarr; View &rarr; Edit &rarr; Advanced Settings
```

Comment:
:rocket: I was not aware that `&rarr;` would display and correctly. But it seems so.